frontend/src/app/supportAdmin/ManageFacility/FacilityInformation.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from "react";
+
+import Button from "../../commonComponents/Button/Button";
+import Modal from "../../commonComponents/Modal";
+import { OrganizationTypeEnum } from "../../signUp/Organization/utils";
+
+import { ManageFacilityState } from "./ManageFacility";
+
+type FacilityDataItemProps = {
+  title: string;
+  description: string;
+};
+
+const FacilityDataItem: React.FC<FacilityDataItemProps> = ({
+  title,
+  description,
+}) => (
+  <div className="margin-y-3">
+    <h4 className="margin-y-1">{title}</h4>
+    <p className="margin-y-1">{description}</p>
+  </div>
+);
+
+export interface FacilityInformationProps {
+  onFacilityDelete: () => void;
+  manageFacilityState: ManageFacilityState;
+}
+
+const FacilityInformation: React.FC<FacilityInformationProps> = ({
+  manageFacilityState,
+  onFacilityDelete,
+}) => {
+  const OrgTypeDictionary = OrganizationTypeEnum as any;
+
+  /**
+   * Confirmation modal
+   */
+  const [showModal, setShowModal] = useState(false);
+  const closeModal = () => setShowModal(false);
+
+  const handleConfirmDeletion = () => {
+    onFacilityDelete();
+    closeModal();
+  };
+
+  const confirmationModal = (
+    <Modal
+      onClose={closeModal}
+      title="Delete facility"
+      contentLabel="Confirm delete facility"
+      showModal={showModal}
+    >
+      <Modal.Header
+        styleClassNames={"font-heading-lg margin-top-0 margin-bottom-205"}
+      >
+        Delete {manageFacilityState.facility?.name}
+      </Modal.Header>
+      <div className="border-top border-base-lighter margin-x-neg-205"></div>
+      <p className="margin-top-4">
+        Deleting {manageFacilityState.facility?.name} will remove any
+        in-progress tests from the test queue. Users only added to this facility
+        will lose access to SimpleReport until an organization admin reassigns
+        them. Patients only added to this facility also need to be reassigned.
+      </p>
+      <p className="margin-top-3">
+        There are{" "}
```

Comment:
can we make sure the copy for "users" and "patients" will make sense for 1 user and 1 patient

Reply:
I think it should also say "There is 1 user" for the single user case.

Reply:
Fixed!